### PR TITLE
Add interaction metadata for export and investment advice topics

### PR DESCRIPTION
**Pull request by elcct**
### Description of change

This adds export and investment advice topics metadata and Django admin to manage the tables. 
Migration contains lists of initial topics.
It also adds `expert_advice_topic` and `investment_advice_topic` fields to interactions, so that interaction can have a one topic. Many to many relation was considered not necessary.

PRs with updated views, validation etc. will come soon.

Some admin screens:

admin1
admin2
admin3
admin4

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [x] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/interaction/admin.py [hunk 1]
```diff
@@ -18,16 +18,19 @@
...
     InteractionPermission,
+    InvestmentAdviceTopic,
     PolicyArea,
     PolicyIssueType,
 )
 from datahub.metadata.admin import DisableableMetadataAdmin, OrderedMetadataAdmin
 
 admin.site.register(CommunicationChannel, DisableableMetadataAdmin)
 admin.site.register((PolicyArea, PolicyIssueType), OrderedMetadataAdmin)
+admin.site.register((ExportAdviceTopic, InvestmentAdviceTopic), OrderedMetadataAdmin)
```
Comment (currycoder) [inline datahub/interaction/admin.py]: Could this and the preceeding line be combined?

Comment (elcct) [reply datahub/interaction/admin.py]: I bundled these together now. Thanks!

File: datahub/interaction/models.py [hunk 2]
```diff
@@ -255,6 +278,24 @@ class Interaction(ArchivableModel, BaseModel):
     )
     policy_feedback_notes = models.TextField(blank=True, default='')
 
+    # Export and investment advice topics
+    export_advice_topic = models.ForeignKey(
+        'ExportAdviceTopic',
+        blank=True,
+        related_name='%(class)ss',
+        null=True,
+        on_delete=models.CASCADE,
```
[the remark this answers is not in the document]
Comment (elcct) [reply datahub/interaction/models.py]: Oops. Thanks for spotting this, obviously this is wrong! I fixed it now.

**Comment by marcofucci** [comment]
@elcct sorry I didn't give my feedback but I overheard that there were further discussions related to this PR.

From my point of view, if would be good to have some extra context on how the bigger feature will work and the steps you are going to take to get there as without that it's hard to understand this small piece.

Thanks!

**Comment by elcct** [comment]
The approach to this has changed, so I am closing this for now.

**Pull request closed without merging**